nameserver/nameserver.go

```diff
@@ -137,6 +138,19 @@ func (n *Nameserver) ContainerDied(ident string) {
 	n.broadcastEntries(entries...)
 }
 
+func (n *Nameserver) ContainerDisconnected(ident string) {
+	n.Lock()
+	entries := n.entries.tombstone(n.ourName, func(e *Entry) bool {
+		if e.ContainerID == ident {
+			n.infof("container %s disconnected; tombstoning entry %s", ident, e.String())
```

Comment:
Is it possible for this to happen? It seems that docker stops this from happening.